lib/ansible/parsing/mod_args.py

```diff
@@ -179,7 +179,7 @@ def _normalize_parameters(self, thing, action=None, additional_args=None):
             for arg in args:
                 arg = to_text(arg)
                 if arg.startswith('_ansible_'):
-                    raise AnsibleError("invalid parameter specified for action '%s': '%s'" % (action, arg))
+                    raise AnsibleError("Parameters beginning with '_ansible' are not allowed for the action '%s': '%s'" % (action, arg))
```

Comment:
I would add `The '_ansible_' namespace is reserved for internal engine use.`

Reply:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment) and unit test for this change. See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.